src/components/VtkTwoView.vue

```diff
@@ -362,6 +363,10 @@ export default defineComponent({
       viewProxy.value.getInteractorStyle2D().removeAllManipulators();
     });
 
+    onUnmounted(() => {
+      viewConfigStore.removeView(viewID.value);
```

Comment:
I think VtkThreeView needs this as well.

Reply:
Added